server/src/domain/types.rs

```diff
@@ -378,16 +392,20 @@ impl IntoActiveValue<Serialized> for JpegPhoto {
     }
 }
 
+#[derive(PartialEq, Eq, Debug, Clone, Serialize, Deserialize)]
+pub struct AttributeValue {
+    pub name: String,
+    pub value: Serialized,
+}
+
 #[derive(PartialEq, Eq, Debug, Clone, Serialize, Deserialize)]
 pub struct User {
     pub user_id: UserId,
     pub email: String,
     pub display_name: Option<String>,
-    pub first_name: Option<String>,
```

Comment:
Meat of the diff: Removing these attributes in the `User` class, the values are stored in the `attributes` vector instead.